run_xnat2bids.py

```diff
@@ -331,15 +361,106 @@ async def main():
             "command": sbatch_cmd_without_password
         })
         
-        # Run xnat2bids asynchronously.
-        task = asyncio.create_task(asyncio.create_subprocess_exec(*sbatch_cmd))
-        tasks.append(task)
+        # Run xnat2bids 
+        proc = await asyncio.create_subprocess_exec(*sbatch_cmd, stdout=asyncio.subprocess.PIPE)
+
+        stdout, stderr = await proc.communicate()
+        output.append(stdout)
+    
+    return output, needs_validation
+
+async def launch_bids_validator(arg_dict, user, password, bids_root, job_deps):    
+
+    # Establish connection 
+    connection = requests.Session()
```

Comment:
I think we should be closing these connections

Reply:
Done!